package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe-redirects",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Update the changelog (end of the README)

Reply:
good call.